frontend/src/app/page.tsx

```diff
@@ -16,15 +17,32 @@ export default function Home() {
     getVids();
   }, []);
 
+  useEffect(() => {
+    const getProgress = async () => {
+      if (!videos.length) return
+      const inProgressVideoIds = videos.filter(video => video.processing).map(video => video.id).join('&cid=')
+      const resp = await fetch(`/api/clips/progress?cid=` + inProgressVideoIds);
+      const {clips} = (await resp.json()) as Progress
+      setVideoProgresses(clips)
+    }
+    const interval = setInterval(getProgress, 1000);
```

Comment:
Have we thought about how often we want to do this check? I don't have any particular problems with 1 second, but it feels a little quick.

Reply:
Could pump it up to 2 or 3 if need be.

Reply:
I think 2 seconds would be a good number after we have animations on the progress bar. I mostly chose 1 second for the upload page because and higher and the bar would snap too much visually when updating